src/notification.scss

```diff
@@ -29,315 +29,140 @@ $block: #{$fd-namespace}-notification;
 
+  @mixin fd-priority-indicator($type) {
     @include fd-rtl() {
-      padding: 1rem 1rem 0 0.5rem;
+      padding-right: 0.5rem
     }
-
     @include fd-icon("message-#{$type}");
   }
 
-  // ------------------------ CONTAINER ------------------------
-  @include fd-reset();
-
-  max-width: $fd-notification-max-width;
   background-color: $fd-notification-background-color;
   box-shadow: $fd-notification-box-shadow;
   border-radius: $fd-notification-border-radius;
-  border-left: 0.375rem solid;
-  border-left-color: $fd-notification-indicator-border-color-default;
-  margin-bottom: 0.5rem;
-
-  @include fd-rtl() {
-    border-left: none;
-    border-right: 0.375rem solid;
-    border-right-color: $fd-notification-indicator-border-color-default;
-  }
+  padding: 1rem 0.5rem 1rem 1rem;
+  margin-bottom: 20px;
+  width: 100%;
+  max-width: $fd-notification-max-width-m;
+  box-sizing: border-box;
 
-  // ------------------------ HEADER ------------------------
-  &__header {
+  &__body {
     @include fd-reset();
-
     display: flex;
-    box-shadow: inset 0 -0.0625rem $fd-notification-header-border-color;
-  }
-
-  &__indicator {
-    @include fd-reset();
-
-    &--error {
-      @include fd-priority-indicator('error');
-
-      color: $fd-notification-indicator-color-error;
-    }
-
-    &--success {
-      @include fd-priority-indicator('success');
-
-      color: $fd-notification-indicator-color-success;
-    }
-
-    &--warning {
-      @include fd-priority-indicator('warning');
-
-      color: $fd-notification-indicator-color-warning;
-    }
-
-    &--information {
-      @include fd-priority-indicator('information');
+    justify-content: space-between;
+    padding: 0;
 
-      color: $fd-notification-indicator-color-information;
+    &.message {
+      padding-top: 1rem;
     }
   }
 
-  &__title {
-    @include fd-reset();
-
-    color: $fd-notification-title-color;
-    padding: 1rem 0;
-    margin-right: 1rem;
-    font-weight: 700;
-    width: 100%;
+  &.notification-group {
+    padding: 0;
 
-    @include fd-rtl() {
-      margin-right: 0;
-      margin-left: 1rem;
-    }
-
-    &:first-child {
-      padding-left: 1rem;
+    .#{$block}__body {
+      padding: 1rem 0.5rem 1rem 1rem;
+      border-bottom: $fd-notification-border;
 
-      @include fd-rtl() {
-        padding-left: 0;
-        padding-right: 1rem;
+      &:last-child {
+        border-bottom: none;
       }
     }
-  }
-
-  &__close {
-    position: relative;
-    top: 0.45rem;
-    right: 0.45rem;
 
-    @include fd-rtl() {
-      right: -0.45rem;
+    .#{$block}__group-header {
+      display: flex;
+      align-items: baseline;
+      padding: 0.75rem 0.5rem 0 1rem;
+      border-bottom: $fd-notification-border;
     }
   }
 
-  // ------------------------ BODY ------------------------
-  &__body {
-    @include fd-reset();
-
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
+  .fd-avatar {
+    margin-right: 0.75rem;
```

Comment:
RTL support missing

Reply:
Add margin to `fd-notification__content` and remove this instruction. Try to avoid modifying other components